moveit_core/package.xml

```diff
@@ -68,6 +68,7 @@
   <test_depend>orocos_kdl</test_depend>
   <test_depend>ament_cmake_gtest</test_depend>
   <test_depend>ament_index_cpp</test_depend>
+  <test_depend>ros_testing</test_depend>
```

Comment:
I don't see where it's being used in moveit_core, I think we should remove [this line](https://github.com/ros-planning/moveit2/blob/main/moveit_core/online_signal_smoothing/CMakeLists.txt#L49) rather than adding the dependency here
```suggestion
```

Reply:
Good catch, thank you.  I pushed this change instead.